mailbox/api/src/main/java/org/apache/james/mailbox/model/MailboxAnnotationKey.java

```diff
@@ -31,16 +31,20 @@
 
```

Comment:
This change is related to the RFC. Could you indicate the section that motivate these changes ?
(So that I do not need to look for it myself)
It will makes review way easier as we would know it is not something coming from nowhere